mappings/net/minecraft/server/MinecraftServer.mapping

```diff
@@ -129,7 +129,16 @@ CLASS net/minecraft/server/MinecraftServer
 	METHOD method_3718 isFlightEnabled ()Z
 	METHOD method_3719 getGameProfileRepo ()Lcom/mojang/authlib/GameProfileRepository;
 	METHOD method_3723 save (ZZZ)Z
+		COMMENT Saves the server to the data storage device.
+		COMMENT
+		COMMENT To store the player data in addition to server data, call {@link PlayerManager#saveAllPlayerData()}.
+		COMMENT
+		COMMENT @return Whether saving was successful.
 		ARG 1 suppressLogs
+		ARG 2 flush
+		    COMMENT Immediately write all data to storage device
+		ARG 3 force
+		    COMMENT When set to true all the {@link ServerWorld}s will be saved even if {@link ServerWorld#savingDisabled} is set to true.
```

Comment:
```suggestion
		    COMMENT when set to true, all the {@link ServerWorld}s will be saved even if {@link ServerWorld#savingDisabled} is set to true
```

Reply:
I believe that sentences should start with a capital letter and end with a full stop.

Reply:
That is not the Yarn convention.